ta/mk/ta_dev_kit.mk

```diff
@@ -78,6 +78,10 @@ endif
 libnames += dl
 libdeps += $(ta-dev-kit-dir$(sm))/lib/libdl.a
 
+# libutils provides __getauxval needed by libgcc
```

Comment:
Sorry for commenting again...
This doesn't tell why it is not OK to simply move libutils after libgcc, in other words: why libutils needs to be before *and* after. I think more details would be good in case someone wants to touch this later -- mentioning the issues you met with GCC 10 would help I think.

Reply:
Agree. I elaborated the comment, mentioning both version of libgcc and dependency issues.

Reply:
Excellent, thank you :+1: